gnomad/utils/vcf.py

```diff
@@ -796,8 +878,93 @@ def make_info_dict(
             },
         }
         info_dict.update(popmax_dict)
+    if grpmax:
+        grpmax_dict = {
+            f"{prefix}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AC{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Allele count in the genetic ancestry group with the maximum allele"
+                    f" frequency{description_text}"
+                ),
+            },
+            f"{prefix}AN{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Total number of alleles in the genetic ancestry group with the"
+                    f" maximum allele frequency{description_text}"
+                ),
+            },
+            f"{prefix}AF{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Maximum allele frequency across genetic ancestry"
+                    f" groups{description_text}"
+                ),
+            },
+            f"{prefix}nhomalt{label_delimiter}grpmax{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Count of homozygous individuals in the genetic ancestry group"
+                    f" with the maximum allele frequency{description_text}"
+                ),
+            },
+        }
+        info_dict.update(grpmax_dict)
 
-    else:
+    if fafmax:
+        fafmax_dict = {
+            f"{prefix}fafmax{label_delimiter}faf95{label_delimiter}max{suffix}": {
+                "Number": "A",
+                "Description": (
+                    "Maximum filtering allele frequency (using Poisson 95% CI)"
+                    f" across genetic_ancestry groups{description_text}"
+                ),
+            },
+            f"{prefix}fafmax{label_delimiter}faf95{label_delimiter}max{label_delimiter}gen{label_delimiter}anc{suffix}": {
```

Comment:
Just for consistency, I saw below, you used `data_type`, why this `gen_anc` has to used  `{label_delimiter}`?

Reply:
Yeah I think data type should use the delimiter too -- ill fix it